app/cafe.py

```
@@ -0,0 +1,21 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(visitor.get("name", "Unknown visitor"))
+
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError(visitor.get("name", "Unknown visitor"))
+
+        if not visitor.get("wearing_a_mask", False):
+            raise NotWearingMaskError(visitor.get("name", "Unknown visitor"))
```

This violates checklist item #3: 'Provide descriptive messages when raising custom errors.'

See above for the required message format for not wearing a mask errors.